.eslintrc

```diff
@@ -0,0 +1,78 @@
+ecmaFeatures:
+  ecmascript: 6
+
+env:
+  browser: true
+  node: true
+
+globals:
+  jQuery: true
+  $: true
+  Backbone: true
+  _: true
+
+rules:
+  # Possible errors
+  no-cond-assign: 0
+  no-comma-dangle: 2
+  no-console: 1
+  no-extra-parens: 2
+  no-irregular-whitespace: 2
+  no-reserved-keys: 2
+
+  # Best practices
+  block-scoped-var: 2
+  eqeqeq: [2, smart]
+  no-floating-decimal: 2
+  wrap-iife: [2, inside]
+  curly: 2
+  no-alert: 2
+  no-eval: 2
+  no-extend-native: 2
+  no-with: 2
+
+  # Variables
+  no-use-before-define: 2
+  no-shadow-restricted-names: 2
+
+  # Strict Mode
+  strict: [2, "global"]
```

Comment:
I believe there's a caveat with this in that global mode will actually not work exactly like the docs say if the linter is only checking the source (non-concatenated) JS files—which it should be to avoid issues with vendor scripts. After browserfying, the modules will be wrapped in anonymous functions and the concatenated code will have strict directives that occur multiple times at the function-level. This seems fine to me, but if someone wanted to spend the time there could be a step in minification process that removed the multiple strict directives and wrapped one directive in the global scope of all the browserfy modules (but excluding vendor script, as they could have their own strict mode settings).